arch/common/gen_isr_tables.py

```diff
@@ -143,13 +147,55 @@ def write_source_file(fp, vt, swt, intlist):
         fp.write("\t{(void *)0x%x, (void *)0x%x},\n" % (param, func))
     fp.write("};\n")
 
+def get_symbols(obj):
+    for section in obj.iter_sections():
+        if isinstance(section, SymbolTableSection):
+            return {sym.name: sym.entry.st_value
```

Comment:
Indentation here seems weird in the next line. GitHub glitch?

Reply:
No. It's a glitch in my copy-paste. Will correct it.